<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userUI</class>
 <widget class="QWidget" name="userUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minor Cineprex</string>
  </property>
  <widget class="QLabel" name="greeting">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hello, </string>
   </property>
  </widget>
  <widget class="QLabel" name="cash">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Your Cash:</string>
   </property>
  </widget>
  <widget class="QLabel" name="rank">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>651</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>userBG.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="ticketButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TICKET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>greeting</zorder>
  <zorder>cash</zorder>
  <zorder>rank</zorder>
  <zorder>ticketButton</zorder>
  <zorder>quitButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
